readme.md

```diff
@@ -60,4 +60,4 @@ We use GitHub for version control (https://github.com)
 See also the list of collaborators (https://github.com/piazzaana/chicken-fries/settings/collaboration) who participated in this project.
 
 ## License
-This project is licensed under the GPLv3 License - see the [LICENSE.md](https://github.com/piazzaana/chicken-fries/blob/develop/docs/LICENSE.md) file for details
\ No newline at end of file
+This project is licensed under the GPLv3 License - see the [LICENSE.md](https://github.com/piazzaana/chicken-fries/blob/develop/docs/LICENSE.md) file for details
```

Comment:
Best Practice: This should link to the released version (whats on the master branch) not the ```develop``` branch of the license.

Reply:
Fix it, https://github.com/piazzaana/chicken-fries/issues/60

Reply:
Needs to be actually fixed before approval will be granted.

Reply:
@reactivepixel  I'm not sure why you are not seeing the changes, but this is the address I get when I click on the license link https://github.com/piazzaana/chicken-fries/blob/master/docs/LICENSE.md